flocker/volume/service.py

```diff
@@ -48,8 +48,8 @@ class VolumeName(object):
     :ivar unicode namespace: The namespace of the volume,
         e.g. ``u"default"``. Must not include periods.
 
-    :ivar unicode id: The id of the volume,
-        e.g. ``u"mypostgresdata"``. Since volume ids must match Docker
+    :ivar unicode dataset_id: The unique id of the
+        dataset. ``u"mypostgresdata"``. Since volume ids must match Docker
```

Comment:
The need to document character constraints here is probably a good point in favor of (eventually) representing these as UUID instances instead of unicode.